### ISSUE TITLE
[indent] import require are not reported as error

### ISSUE THREAD

**Issue by ficristo** [eslint/typescript-eslint-parser#577]
**What version of TypeScript are you using?**
3.1.6

**What version of `typescript-eslint-parser` are you using?**
21.0.2

**What code were you trying to parse?**

```ts
// The code:
    import Dialogs              = require("widgets/Dialogs");
```

**What did you expect to happen?**
An error because the code is indented but shouldn't.

For now my rule indent is:
```js
"indent": ["error", 4, {
    "SwitchCase": 1,
    "VariableDeclarator": 1,
    "FunctionDeclaration": { "parameters": "first", body: 1 },
    "FunctionExpression": { "parameters": "first", body: 1 },
    "CallExpression": { "arguments": 1 },
    "ArrayExpression": 1,
    "ObjectExpression": 1,
    "ImportDeclaration": 1,
    "flatTernaryExpressions": false
}],
```

**What happened?**
No error.

**Comment by armano2** [comment]
`TSImportEqualsDeclaration` is not supported by eslint rule, this should be added int https://github.com/bradzacher/eslint-plugin-typescript

**Comment by ficristo** [comment]
It should be fixed by https://github.com/bradzacher/eslint-plugin-typescript/pull/219
